docs/Logos-Hook-Splitting.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Logos: Hook Splitting"
+layout: docs
+---
+
+By default, the Logos pre-processor will only process one .xm file at build time. However, it is possible to split the Logos hooking code into multiple files.
+
+To do this, the main file has to be renamed to an .xmi file. Then, other .xm files can be included in it using the `#include` or `#import` directives. The Logos pre-processor will add those files to the main file before processing it.
```

Comment:
However, those other .xm files have to not be included in the list of files to compile. We have to specify that XXX_FILES has to contain the .xmi that includes/imports the other files and none of those have to be in the XXX_FILES.

Reply:
Good call!